<!DOCTYPE html>
<html lang="en-US">

<head>
	<title>Unofficial Resident to Resident Guide</title>
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<style>
	html * {
		line-height: 1.625;
		color: #757575;
		font-family: sans-serif;
	}

	img {
		display: block;
		margin-left: auto;
		margin-right: auto;
	}

	.a {
		color: #007360
	}

	</style>
</head>

<body style="background-color:Black;">

	<img src="images/image1.png" class="center" width="200" height="200"> 
	<h1 style="text-align: center; font-size: 200%;">Unofficial Molecular Resident-to-Resident Guide</h1>

	<p style="text-align: center;"><b>Disclaimer:</b> This is an unofficial resource. When in doubt, ask the attending. If something is wrong/inaccurate, <a href="https://github.com/sb-ml-py/Unofficial-Molecular-Guide/discussions" class = "a">let us know.</a></p>

	<br><br><hr><br><br>
	<h2 style="text-align: center;">Content</h2>

		<p>
		<a href="https://gowustl.sharepoint.com/sites/MolecularRotation/SitePages/People.aspx" class = "a" style="margin-left: 40px;">Molecular Faculty and Staff</a><br>
		<a href="#Resident-Expectations" class = "a" style="margin-left: 40px;">Resident Expectations</a><br>
		<a href="#Navigating-Sharepoint" class = "a" style="margin-left: 40px;">Navigating Sharepoint</a><br>
		<a href="#AutoHotKey" class = "a" style="margin-left: 40px;">AutoHotkey</a><br>
		<a href="#Molecular-Diagnostics-Lab" class = "a" style="margin-left: 40px;">Molecular Diagnostics Lab (MDL)</a><br>
			<a href="#MDL-Call" class = "a" style="margin-left: 80px;">MDL Call</a><br>
			<a href="#Hematologic-Molecular-Algorithm" class = "a" style="margin-left: 80px;">Hematologic Molecular Algorithm (HMA)</a><br>
				<a href="#Chromoseq" class = "a" style="margin-left: 120px;">Chromoseq</a><br>
				<a href="#Clonoseq" class = "a" style="margin-left: 120px;">Clonoseq</a><br>
				<a href="#FLT3" class = "a" style="margin-left: 120px;">FLT3</a><br>
				<a href="#Myeloseq" class = "a" style="margin-left: 120px;">Myeloseq</a><br>
				<a href="#Tempus" class = "a" style="margin-left: 120px;">Tempus</a><br>
			<a href="#Molecular-Sendout-Testing" class = "a" style="margin-left: 80px;">Moleclar Sendout Testing (alphabetical order)</a><br>
				<a href="#ABL-kinase-domain" class = "a" style="margin-left: 120px;">ABL Kinase Domain Mutation</a><br>
				<a href="#Apolipoprotein-E" class = "a" style="margin-left: 120px;">Apolipoprotein E</a><br>
				<a href="#Apolipoprotein-L1" class = "a" style="margin-left: 120px;">Apolipoprotein L1</a><br>
				<a href="#CAKUT" class = "a" style="margin-left: 120px;">Congenital Anomalies of the Kidney and Urinary Tract Panel</a><br>
				<a href="#CYP450" class = "a" style="margin-left: 120px;">CYP450 family sequencing</a><br>
				<a href="#DPYD-Testing" class = "a" style="margin-left: 120px;">DPYD Testing</a><br>
				<a href="#FSHD1" class="a" style="margin-left: 120px;">FSHD1 Southern Blot Test</a><br>
				<a href="#GenomeXpress" class="a" style="margin-left: 120px;">GenomeXpress</a><br>
				<a href="#Huntington" class = "a" style="margin-left: 120px;">Huntington Disease Genetic Testing</a><br>
				<a href="#KITVS" class="a" style="margin-left: 120px;">KIT Asp816Val Mutation Analysis</a><br>
				<a href="#TPTKFV" class="a" style="margin-left: 120px;">Targeted Prenatal Testing of Known Familial Variants</a><br>
				<a href="#Telomere" class="a" style="margin-left: 120px;">Telomere Length Measurement</a><br>
				<a href="#VHL" class="a" style="margin-left: 120px;">Von-Hippel-Lindau Gene Sequencing</a><br>
				<a href="#XomeDx" class = "a" style="margin-left: 120px;">XomeDxXpress Exome Sequencing</a><br>
		<a href="#Next-Generation-Sequencing" class = "a" style="margin-left: 40px;">Next Generation Sequencing (NGS)</a><br>
			<a href="#NGS-Myeloseq" class = "a" style="margin-left: 80px;">Myeloseq</a><br>
			<a href="#NGS-Chromoseq" class = "a" style="margin-left: 80px;">Chromoseq</a><br>
			<a href="#Cytogenetics" class = "a" style="margin-left: 80px;">Cytogenetics</a><br>
			<a href="#NGS-Gatewayseq" class = "a" style="margin-left: 80px;">Gatewayseq</a><br>
		</p>

	<br><br><hr><br><br>
	<h1 style="text-align: center;">General Guidance<h1>


	<h2 id="Resident-Expectations">Resident Expectations</h2>

		<h3>MDL Call</h3>
			<p>
			<ul>
			<li>Hold and answer call phone 8am-5pm.</li>
			<li>Manage Molecular Sendout Log (see MDL call section)</li>
			</ul>
			</p>

		<h3>Myeloseq & Chromoseq</h3>
			<p>
			<ul>
			<li>Complete assigned Myeloseq and Chromoseq interpretations before signout each week.</li>
			<li>Myeloseq and Chromoseq signout are at 2pm on Thursday.</li>
			<li>Cases will be assigned Monday or Tuesday and become available for interpretation Wednesday.</li>
			</ul>
			</p>

		<h3>Presentations</h3>
			<p>
			<ul>
			<li><u>LGM Case conference:</u> One 25 min presentation in front of all LGM. Focus on clinical case, diagnosis, and lit review.</li>
			<li><u>Heme Genomics Conference:</u> One 25min presentation in front of Hematopathology and Molecular divisions. Focus on hematopathology topic.</li>
			<li><u>Tuesday Molecular Didactic:</u> Two up-to-1-hour presentations in front of the molecular trainees. The first topic will be assigned. The second you choose from the <a href="https://link.springer.com/book/10.1007/978-3-642-19677-5" class="a">DMPP textbook</a>.</li>
			<li><u>NGS Mock Case Presentation:</u> One 10-15 min presentation in front of NGS faculty and molecular trainees. A sample case with germline variants is given, and you explain their meaning according to ACMG guidelines.</li>
			<li><u>Beeper Report:</u> One presentation for each week youâ€™re on MDL call. Discussion of new acute leukemias (AML/ALL), new acute leukemia relapses, and non-hematologic sendout testing.</li>
			</ul>
			</p>
			
		<h3>Non-Clinical Requirements</h3>
			<p>
			<ul>
			<li>Arrange 1-week when you're not on call to rotate in the cytogenetics lab. To schedule, email dr. Yang Cao.</li>
			<li>Complete <a href="https://gowustl.sharepoint.com/:x:/r/sites/MolecularRotation/_layouts/15/Doc.aspx?sourcedoc=%7B0A60D366-0E58-4D9D-8D53-4A1ADCA0E7EC%7D&file=MolecularResFellowTrainingLog_2022.xlsx&action=default&mobileredirect=true&DefaultItemOpen=1" class = "a">MDL Wet Lab Worksheet</a>. Have lab techs sign off on these tasks.</li>
			<li>Join the <a href="https://www.amp.org/membership/join-renew/" class="a">Association for Molecular Pathology</a> and Complete the <a href="https://educate.amp.org/local/catalog/view/product.php?productid=309" class="a">NGS101 module</a>.
			<li>Attend LGM and molecular didactics:
				<li style="margin-left: 40px;"><u>Monday:</u> None</li>
				<li style="margin-left: 40px;"><u>Tuesday:</u> 12:00 - LGM Case Conference; 15:00 - Molecular Didactic</li>
				<li style="margin-left: 40px;"><u>Wednesday:</u> 12:00 - RFM/Path management/Digital Pathology Conference</li>
				<li style="margin-left: 40px;"><u>Thursday:</u> 9:30 - Clinical Genomics Conference, 10:30 - MDL Beeper Report, 12:00 - LGM Grand Rounds</li>
				<li style="margin-left: 40px;"><u>Friday:</u> 12:00 - Heme/Genomics Conference</li>
			<li>Required readings (honor system):</li>
				<li style="margin-left: 40px;">Molecular Pathology in Clinical Practice (2016) Chapter 62: Molecular Pathology Lab Management</li>
				<li style="margin-left: 40px;">Modern Clinical Molecular Techniques (2012) Chapter 3: Test Validation</li>
				<li style="margin-left: 40px;">Mastering Genomic Terminology</li>
				<li style="margin-left: 40px;">Words Matter: The Language of Difference in Human Genetics</li>
				<li style="margin-left: 40px;">ACMG Constitutional Variant Guidelines</li>
				<li style="margin-left: 40px;">AMP/ASCO/CAP Somatic Variant Guidelines</li>
			</ul>
			</p>

	<h2 id="Navigating-Sharepoint">Navigating Sharepoint</h2>
		<p>It&#8217;s chaos y&#8217;all. Use the search bar.</p>
	
	<h2 id="AutoHotKey">AutoHotKey</h2>
		<p>
		AutoHotKey is a program that the surgical pathology residents and attendings use for quick-texts. It auto-populates things you write a lot, and can save time. Either get one of them to send it to you and show you how to set it up, <a href="https://github.com/sb-ml-py/Pathology-AutoHotKey" class="a">set it up from my guide</a>, or <a href="https://www.autohotkey.com/" class="a">set it up from scratch</a>. Unless you got it from another resident, then the specific molecular templates we use won't be in there. They are listed below. Just copy them into the Molecular section of your <span style="font-family: courier; color: #514D43;">.ahk</span> file.<br>
		</p>
		
		<ul>
		<li style="font-family: courier; color: #514D43;">;Myeloseq</li>
		<li style="font-family: courier; color: #514D43;">:*:mclin\::[xx]-year-old [xx] with a history of [xx] (diagnosed YYYY-MM-DD) with [cytogenetic abnormalities], who is status post hematopoietic cell transplant (YYYY-MM-DD) and presents for [evaluation|treatment|further workup]. Corresponding bone marrow biopsy (SXX-XXXXX) shows [xx]. Corresponding cytogenetic studies (GXX-XXXXX) show [list of FISH findings]. Corresponding ChromoSeq study (WXX-XXXXX) showed [list of variants].</li>
		<li style="font-family: courier; color: #514D43;">:*:mnew\::Mutations in [xx] are identified. This case is best classified as [xx] by ICC criteria or [xx] by the 5th edition World Health Organization Classification of Haematolymphoid Tumours (WHO-HAEM5). The overall constellation of mutations is consistent with [xx] by National Comprehensive Cancer Network (NCCN) guidelines. Overall quality control metrics are within normal limits.</li>
		<li style="font-family: courier; color: #514D43;">:*:mrep\::Previously identified mutations in [xx] are again seen, consistent with persistent molecular disease (measurable residual disease (MRD) positivity). Additional new mutations in [xx] are identified. The previously identified mutations in [xx] are not seen. [chimerism] Overall quality control metrics are within normal limits.</li>
		<li style="font-family: courier; color: #514D43;">:*:mnewneg\::No mutations are identified. Overall quality control metrics are within normal limits. The absence of a mutation detectable with this assay does not rule out hematopoietic malignancy.</li>
		<li style="font-family: courier; color: #514D43;">:*:mneg\::Previously identified mutations in [xx] are not seen, consistent with mutation clearance and measurable residual disease (MRD) negative status at a sensitivity of at least 0.1%. [chimerism] Overall quality control metrics are within normal limits.</li>
		<li style="font-family: courier; color: #514D43;">;Chromoseq</li>
		<li style="font-family: courier; color: #514D43;">;Gatewayseq</li>
		</ul>
	
	<h1 id="Molecular-Diagnostics-Lab" style="text-align: center;">Molecular Diagnostics Lab (MDL)</h1>

		<h2 id="MDL-Call">MDL Call</h2>
			<h3>Managing the <a href="https://path.wustl.edu/sendoutlog/Trainee/MyWorklist" class = "a">Sendout Log </a></h3>
				<p>
				New cases appear in the &#8216;&#8216;Pending&#8217;&#8217; tab. You should click them and hit [update] button to add them to your queue. Your queue is in the &#8216;&#8216;My Worklist&#8217;&#8217; tab. Organize your worklist by sorting the cases by &#8216;&#8216;Id.&#8217;&#8217; You should address new cases 2-3 times per day and run the entire Worklist daily to check for updates. Before closing a case, if there are relevant biopsies or cytogenetics/FISH in CoPath, the attendings like these results recorded in the log. Tests that were already auto-approved do not bear thinking about, just hit the &#8216;&#8216;approve&#8217;&#8217; button and move on. It is normal for cases to drag out for several days awaiting bone marrow and FISH/cytogenetic results.<br>
				<br>
				Cases will fall into three categories: Hematologic molecular algorithm (HMA), hematologic non-HMA, and non-hematologic.<br>
				<ul>
				<li><u>HMA cases</u> are characterized by an HMA order. follow the instructions in the HMA for them.</li><br>
				<li><u>Hematologic non-HMA</u> cases are characterized by hematologic orders such as FLT3 or Myeloseq, but no HMA order. For these you should approve if indicated by the HMA, or if not indicated by the HMA, ask the clinicians what they&#8217;re looking for and whether they&#8217;d like to change to testing indicated by the HMA. If there is no HMA order, you cannot add additional testing, so if the clinician wants it, they must order it.</li><br>
				<li><u>Non-hematologic cases</u> involve miscellaneous sendout tests. These are similar to test approvals from chemistry and micro, but for the fact that we rarely say &#8216;&#8216;no.&#8217;&#8217; Information on what you should do with these may be found herein (<a href="#Molecular-Sendout-Testing" class = "a">Moleclar Sendout Testing (alphabetical order)</a>) or if you search historical cases in the call log.</li><br>
				</ul>
				</p>



		<h2 id="Hematologic-Molecular-Algorithm">Hematologic Molecular Algorithm (HMA)</h2>

			<h3 id="Chromoseq">Chromoseq</h3>

			<h3 id="Clonoseq">Clonoseq</h3>

			<h3 id="FLT3">FLT3</h3>

			<h3 id="Myeloseq">Myeloseq</h3>

			<h3 id="Tempus">Tempus</h3>



		<h2 id="Molecular-Sendout-Testing">Molecular Sendout Testing</h2>

			<h3 id="ABL-kinase-domain">ABL Kinase Domain Mutation</h3>
				<p>
				<u>Testing laboratory:</u> MAYO<br>
				<u>Test ID:</u> BAKDM<br>
				<u>Test Webpage:</u>  <a href="https://www.mayocliniclabs.com/test-catalog/Overview/89609" class = "a">Link</a><br>
				<u>Indication:</u> This test is indicated in Philadelphia Chromosome (BCR::ABL) positive patients who are failing treatment with imatinib, when the clinical team is planning to treat with the alternative drug, asciminib.<br>
				</p>
				
			<h3 id="Apolipoprotein-E">Apolipoprotein E</h3>
				<p>
				<u>Testing laboratory:</u> MAYO<br>
				<u>Test ID:</u> APOEG<br>
				<u>Test Webpage:</u>  <a href="https://www.mayocliniclabs.com/test-catalog/Overview/35358" class = "a">Link</a><br>
				<u>Indication:</u> Appropriate in individuals undergoing treatment or planning to undergo treatment with lecanemab.<br>
				</p>

			<h3 id="Apolipoprotein-L1">Apolipoprotein L1</h3>
				<p>
				<u>Testing laboratory:</u> MAYO<br>
				<u>Test ID:</u> APOL1<br>
				<u>Test Webpage:</u>  <a href="https://www.mayocliniclabs.com/test-catalog/Overview/605251" class = "a">Link</a><br>
				<u>Indication:</u> Variants associated with development of kidney disease. Appropriate in African americans undergoing kidney disease workup or in individuals who will be kidney donors.<br>
				</p>

			<h3 id="CAKUT">Congenital Anomalies of the Kidney and Urinary Tract Panel</h3>
				<p>
				<u>Testing laboratory:</u> Invitae<br>
				<u>Test ID:</u> 434341<br>
				<u>Test Webpage:</u>  <a href="https://www.invitae.com/en/providers/test-catalog/test-434341" class = "a">Link</a><br>
				<u>Indication:</u> This test analyzes 41 genes associated with congenital anomalies of the urinary tract. It is indicated if there is clinical concern for a genetic cause of a congenital urinary abnormality.<br>
				</p>
				
			<h3 id="CYP450">CYP450 family sequencing</h3>
				<p>
				<u>Testing laboratory:</u> MAYO<br>
				<u>Test ID:</u> Varies<br>
				<u>Test Webpage:</u> N/A<br>
				<u>Indication:</u> This is a number of associated tests that can be ordered together or individually. They are indicated in patients for which there is a clinical concern for abnormal drug metabolization. It is important to order the test for the correct CYP450 family enzyme which corresponds to the drug of concern. For example, CYP450 2C19R metabolizes voriconazole, so the appropraite test is <a href="https://www.mayocliniclabs.com/test-catalog/Overview/610043#Overview" class="a">MAYO Test ID: 2C19R</a>, while CYP3A4 metabolizes cyclosporine, so the appropriate test would be <a href="https://www.mayocliniclabs.com/test-catalog/Overview/610046" class="a">MAYO Test ID: 3A4Q</a>. If there is clinical concern for multiple CYP450 family abnormalities, multiple tests can be ordered on the same specimen: <a href="https://www.mayocliniclabs.com/-/media/it-mmfiles/Special%20Instructions/4/3/D/Multiple_Genotype_Test_List" class="a">Link</a>.<br>
				</p>

			<h3 id="DPYD-Testing">DPYD Testing</h3>
				<p>
				<b>DPYD Genotyping</b>
				<u>Testing laboratory:</u> MAYO<br>
				<u>Test ID:</u> DPYDQ<br>
				<u>Test Webpage:</u>  <a href="https://www.mayocliniclabs.com/test-catalog/Overview/610052" class = "a">Link</a><br>
				<u>Indication:</u> Variants in DPYD may cause impaired metabolization of 5-fluorourocil (5-FU) or capecitabine. This test detects eight common variants that cause reduced DPYD activity. These variants account for approximately 98% of clinically significant variants. It is appropriate if clinicians are planning to treat with 5-FU or capecitabine and want to determine proper dosage. There is no significant clinical difference between DPYDQ and DPYDZ so testing can be left to clinician preference.<br>
				</p>
				
				<p>
				<b>DPYD Full Gene Sequencing</b>
				<u>Testing laboratory:</u> MAYO<br>
				<u>Test ID:</u> DPYDZ<br>
				<u>Test Webpage:</u>  <a href="https://www.mayocliniclabs.com/test-catalog/overview/610051" class = "a">Link</a><br>
				<u>Indication:</u> Variants in DPYD may cause impaired metabolization of 5-fluorourocil (5-FU) or capecitabine. Compared to genotyping (DPYDQ), this test detects a broader range of potential variants and is offered at a similar price. It is appropriate if clinicians are planning to treat with 5-FU or capecitabine and want to determine the appropriate dosage. There is no significant clinical difference between DPYDQ and DPYDZ so testing can be left to clinician preference.<br>
				</p>

			<h3 id="FSHD1">FSHD1 Southern Blot Testing</h3>
				<p>
				<u>Testing laboratory:</u> Athena Diagnostics<br>
				<u>Test ID:</u> 405<br>
				<u>Test Webpage:</u>  <a href="https://www.athenadiagnostics.com/view-full-catalog/f/fshd-southern-blot-test" class = "a">Link</a><br>
				<u>Indication:</u> This test may be indicated in patients with clinical concern for fascioscapulohumeral muscular dystrophy. Fascioscapulohumeral muscular dystrophy is characterized by a slowly progressive asymmetric wasting of muscles of the face, shoulder and upper arms.<br>
				</p>
				
			<h3 id="GenomeXpress">GenomeXpress</h3>
				<p>
				<u>Testing laboratory:</u> GeneDX<br>
				<u>Test ID:</u> TH78<br>
				<u>Test Webpage:</u>  <a href="https://providers.genedx.com/tests/detail/genomexpress-trio-rapid-genome-sequencing-with-provisional-results-in-7-days-1146" class = "a">Link</a><br>
				<u>Indication:</u> This test is a rapid whole genome sequencing with interpretation. It may be indicated if there is a high suspicion of genetic disease for which a rapid turn around time would change patient management. Consult service attending before approval.<br>
				</p>
				
	
			<h3 id="Huntington">Huntington Disease Genetic Testing</h3>
				<p>
				<u>Testing laboratory:</u> MAYO<br>
				<u>Test ID:</u> HAD<br>
				<u>Test Webpage:</u>  <a href="https://www.mayocliniclabs.com/test-catalog/Overview/35452" class = "a">Link</a><br>
				<u>Indication:</u> This test may be indicated in patients with a family history of Huntington's Disease. The result can be life altering for patients, so the resident's role is to consult with the clinical team to ensure they have recieved appropriate counseling regarding what the result of this testing may mean. Make sure consent and counseling are documented in EPIC (media tab). You can approve if consent/counseling is documented in EPIC and the testing is ordered by one of the approved neurodegenerative specialists (listed below). Otherwise contact Dr. Parikh.<br>
				<ul>
				<li>Dr. Kevin J Black</li>
				<li>Dr. Isabel Alfradique-Dunham</li>
				</ul>
				</p>
			
			<h3 id="KITVS">KIT Asp816Val Mutation Analysis</h3>
				<p>
				<u>Testing laboratory:</u> MAYO<br>
				<u>Test ID:</u> KITVS<br>
				<u>Test Webpage:</u>  <a href="https://www.mayocliniclabs.com/test-catalog/Overview/607981" class = "a">Link</a><br>
				<u>Indication:</u> This test detects the most common mutation in systemic mastocytosis. It can be subject to false negative in peripheral blood specimens if there are too few circulating mast cells (&lt;0.1%). The test may be indicated in patients for which there is clinical concern for systemic mastocytosis, such as those with otherwise unexplained hives, flushing, anaphylaxis, diarrhea, and lymphadenopathy. There is a low threshold for approval.<br>
				</p>
				
			<h3 id="TPTKFV">Targeted Prenatal Testing for known Familial Variants</h3>
				<p>
				<u>Testing laboratory:</u> Prevention Genetics<br>
				<u>Test ID:</u> N/A<br>
				<u>Test Webpage:</u>  <a href="https://www.preventiongenetics.com/ClinicalTesting/TestCategory/targetedTesting" class = "a">Link</a><br>
				<u>Indication:</u> This test is indicated in patients with a known familial variant and if a family member has previously done this testing, it can be offered at no cost.<br>
				</p>
				
			<h3 id="Telomere">Telomere Length Measurement</h3>
				<p>
				<u>Testing laboratory:</u> RepeatDx (Vancouver)<br>
				<u>Test ID:</u> N/A<br>
				<u>Test Webpage:</u>  <a href="https://repeatdx.com/" class = "a">Link</a><br>
				<u>Indication:</u> Specimen must be recieved by reference lab within 48-hours of collection, by extension it must be received in our lab on or before Wednesday. Indicated for dyskeratosis congenita, idiopathic pulmonary fibrosis, and possibly other genetic disease. The 2-panel test is based on just lymphocyte/granulocytes while the 6-panel test splits lymphocytes into further subsets. There is no clinical reason to do the 6-panel method, and all clinical guidelines are based on the 2-panel method. Johns Hopkins also performs this testing, and we may begin sending this to them in future to avoid delay in Customs.<br>
				</p>
				
			<h3 id="VHL">Von-Hippel-Lindau Gene Sequencing</h3>
				<p>
				<u>Testing laboratory:</u> MAYO<br>
				<u>Test ID:</u> VHLZZ<br>
				<u>Test Webpage:</u>  <a href="https://www.mayocliniclabs.com/test-catalog/Overview/614589" class = "a">Link</a><br>
				<u>Indication:</u> This test may be indicated in patients with clinical concern for Von-Hippel-Lindau Syndrome. It requires additional patient information. The required form can be found on the test catalog entry on MAYO's website, but is also included here: <a href="http://www.mayocliniclabs.com/it-mmfiles/MC1235-98_Molecular_Genetics-Inherited_Cancer_Syndromes_Patient_Information.pdf" class="a">Link</a>.<br>
				</p>
				
			<h3 id="XomeDx">XomeDxXpress Exome Sequencing</h3>
				<p>
				<u>Testing laboratory:</u> GeneDx<br>
				<u>Test ID:</u> N/A<br>
				<u>Test Webpage:</u>  <a href="https://providers.genedx.com/tests/detail/xomedxxpress-trio-wes-with-a-provisional-result-in-7-days-830" class = "a">Link</a><br>
				<u>Indication:</u> This test may be indicated in patients with clinical concern for genetic disease. Consult service attending prior to approval.<br>
				</p>


	<h1 id="Next-Generation-Sequencing" style="text-align: center;">Next Generation Sequencing (NGS)</h1>

			<h3 id="NGS-Myeloseq">Myeloseq</h3>
				<p>
				Dr. Hughes&#8217; Introduction: <a href="https://hughes-drew.github.io/intro_to_myeloseq.html" class = "a">Link</a><br>
				<br>
				<b>Setup Myeloreporter</b><br>
				<ol>
					<li>Go to <span style="font-family: courier; color: #514D43;">\\pathfiles.wustl.edu/amp/Genomics/Myeloseq_Results/Myeloreporter</span></li>
					<li>Copy <span style="font-family: courier; color: #514D43;">index.html</span> into your browser. <em>Bookmark this link so you can access it quickly in future.</em></li>
				</ol>
				<br>
				<b>Interpret Myeloseq</b><br>
				<ol>
					<li>Open Myeloreporter. <em>You should have it bookmarked.</em></li>
					<li>Click <span style="font-family: courier; color: #514D43;">[Choose File]</span></li>
					<li>Open <span style="font-family: courier; color: #514D43;">\\pathfiles.wustl.edu/amp/Genomics/Myeloseq_Results/[current batch]/TWAO-[MRN]-[Accession number]-[Specimen type]-lib1.json</span></li>
					<li>Check that the testing is valid: At the bottom, make sure that <span style="font-family: courier; color: #514D43;">meanCoverage</span> is &gt;1000.00 and <span style="font-family: courier; color: #514D43;">contaminationEstimate</span> is &lt;0.0700</li>
					<li>Fill out the Variant Interpretation, Clinical History, and Specimen Type fields per Dr. Hughes' guide. <em><a href="#AutoHotKey" class="a">AutoHotkey</a> can improve your efficiency.</em></li>
					<li>Click Generate and download RTF. <em>Repeat the above for all your cases before moving onto the next steps.</em></li>
					<li>Open the <span style="font-family: courier; color: #514D43;">.rtf</span> file in Microsoft Word and correct any formating errors. Make sure there appropriate spaces after gene names.</li>
					<li>Open CoPath, navigate to the <span style="font-family: courier; color: #514D43;">Procedure/Addendum</span> tab, and open the case.</li>
					<li>Change the procedure pathologists to the assigned attending and yourself.</li>
					<li>Click <span style="font-family: courier; color: #514D43;">[Edit text]</span> and paste-with-formating (<span style="font-family: courier; color: #514D43;">Ctrl+Shift+V</span>) the information from Microsoft Word into the appropriate fields in CoPath.</li>
					<li>Save by clicking the <em>purple box button</em> then <span style="font-family: courier; color: #514D43;">Complete/Send to Pathologist</span>.
					<li>Repeat for all cases.</li>
				</ol>
				</p>

			<h3 id="NGS-Chromoseq">Chromoseq</h3>
				<p>
				Dr. Hughes&#8217; Introduction: <a href="https://hughes-drew.github.io/intro_to_chromoseq.html" class = "a">Link</a><br>
				</p>

			<h3 id="Cytogenetics">Cytogenetics</h3>

			<h3 id="NGS-Gatewayseq">Gatewayseq</h3>
				<p>
				Dr. Hughes&#8217; Introduction: <a href="https://hughes-drew.github.io/intro_to_gatewayseq.html" class = "a">Link</a><br>
				</p>
				
	<br><br><hr>
		<p>Document last modified:</p>
		<p id="lastModified"></p>

		<script>
		let text = document.lastModified;
		document.getElementById("lastModified").innerHTML = text;
		</script>
</body>

</html>
